src/testing/code-debugging.md

```diff
@@ -878,49 +913,53 @@ result in asserts and generally interfere with your efforts.
+  (Press <kbd>t</kbd> from `flutter run` to call this command.)
   Search for "RepaintBoundary" to see diagnostics
   on how useful a boundary is.
+
 {% comment %} Feature is not yet added to DevTools:
     Rather than using this flag to dump the render tree
     to a file, view the render tree in the Flutter inspector.
     To do so, bring up the Flutter inspector and select the
-    **Render Tree** tab. {% endcomment %}
-<dt markdown="1">[`debugPaintLayerBordersEnabled`][]</dt>
-<dd markdown="1">PENDING
-<dt markdown="1">[`debugRepaintRainbowEnabled`][]</dt>
-<dd markdown="1">You can enable this flag in the Flutter
-    inspector by selecting the **Highlight Repaints** button.
-    If any static widgets are rotating through the colors of the rainbow
-    (for example, a static header), those areas are candidates for adding
-    repaint boundaries.
-<dt markdown="1">[`debugPrintMarkNeedsLayoutStacks`][]</dt>
-<dd markdown="1">Enable this flag if you're seeing more layouts
-    than you expect (for example, on the timeline, on a profile,
-    or from a `print` statement inside a layout method).
-    Once enabled, the console is flooded with stack traces
-    showing why each render object is being marked dirty for
-    layout. You can use the `debugPrintStack()` method from the
-    `services` library to print your own stack traces on demand,
-    if this kind of approach is useful to you.
-<dt markdown="1">[`debugPrintMarkNeedsPaintStacks`][]</dt>
-<dd markdown="1">Similar to `debugPrintMarkNeedsLayoutStacks`,
-    but for excess painting. You can use the `debugPrintStack()`
-    method from the `services` library to print your own stack
-    traces on demand, if this kind of approach is useful to you.
+    **Render Tree** tab.
+{% endcomment %}
+
+[`debugPaintLayerBordersEnabled`][]
+: Enable this flag if you need to see the boundaries of each layer.
+  When enabled, each layer paints a box around its boundary.
+
+[`debugRepaintRainbowEnabled`][]
+: To enable this flag in the Flutter inspector, click **Highlight Repaints**.
+  If any static widgets are rotating through the colors of the rainbow,
+  consider adding repaint boundaries to those areas.
+
+[`debugPrintMarkNeedsLayoutStacks`][]
+: Enable this flag if you're seeing more layouts
+  than you expect (for example, on the timeline, on a profile,
+  or from a `print` statement inside a layout method).
+  When enabled, the console shows stack traces that explain why each
+  the inspector marked each render object as dirty for layout.
+  You can use the `debugPrintStack()` method from the
+  `services` library to print your own stack traces on demand,
+  if this helps you.
+
+[`debugPrintMarkNeedsPaintStacks`][]
+: Similar to `debugPrintMarkNeedsLayoutStacks`,
+  but for excess painting. You can use the `debugPrintStack()`
+  method from the `services` library to print your own stack
+  traces on demand, if this kind of approach is useful to you.
```

Comment:
Since you're touching this, consider dropping the conditional or putting it first.